core/lib/generators/spree/install/templates/config/initializers/spree.rb

```
@@ -91,5 +91,4 @@
 end
 
 Spree.user_class = <%= (options[:user_class].blank? ? 'Spree::LegacyUser' : options[:user_class]).inspect %>
-# Use a different class for admin users
-# Spree.admin_user_class = 'AdminUser'
+Spree.admin_user_class = <%= (options[:admin_user_class].blank? ? (options[:user_class].blank? ? 'Spree::LegacyUser' : options[:user_class]) : options[:admin_user_class]).inspect %>
```

RuboCop (1.81.7)

[fatal] unterminated string meets end of file
(Using Ruby 3.0 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)